config/initializers/menus.rb

```diff
@@ -44,7 +44,7 @@
             { controller: '/news', project_id: nil, action: 'index' },
             if: Proc.new { User.current.allowed_to?(:view_news, nil, global: true) }
   menu.push :time_sheet,
-            { controller: '/time_entries', project_id: nil, action: 'show' },
+            { controller: '/time_entries', project_id: nil, action: 'index' },
```

Comment:
I talked to @ulferts and @floriank whether this is something we want to have a test for. Something along the lines: When I click the link I should see a page.

They could not agree whether it should be RSpec or a Cucumber test, but both agreed that there should be a test of some kind.

**tl; dr:** Write a spec for that, decide upon cuke vs rspec yourself ^^